src/types/responsive.ts

```diff
@@ -0,0 +1,6 @@
+import { Breakpoint } from './breakpoint';
+
+export type Responsive = {
+  breakpoint: Breakpoint;
+  includesContainer?: Boolean;
```

Comment:
Should include a comment here that our default is to treat `undefined` as `true`?